### DISCUSSION TITLE
[9 Settembre 2022] - Cache

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#47] [Cache]
### Traccia Esercizio

![243738335-d75674a4-b9bb-44fc-abf2-3f5e4ea02f85](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/4cc6f298-2a1f-4cd1-b935-0e5c977b1f69)

**Comment by Divine-Sunderer** [comment]
<img width="3343" alt="9 Settembre 2022  - Cache" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/157913185/dedf43a9-d3bb-4835-b5ac-23224cd46c2a">

( E' la mia prima Cache fatta da solo , non so quanto e se è giusta :c )
